meta-resin-common/recipes-support/resin-vars/resin-vars/resin-vars

```diff
@@ -54,6 +54,7 @@ then
     REGISTRY_ENDPOINT=$(jq --raw-output ".registryEndpoint // empty" $CONFIG_PATH)
     VPN_ENDPOINT=$(jq --raw-output ".vpnEndpoint // empty" $CONFIG_PATH)
     DELTA_ENDPOINT=$(jq --raw-output ".deltaEndpoint // empty" $CONFIG_PATH)
+    NEW_HOSTNAME=$(jq --raw-output ".hostname // empty" $CONFIG_PATH)
```

Comment:
On a resin device HOSTNAME is the current  hostname. We can use the same variable, but  I thought this will avoid weird component interaction.